packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -333,16 +336,16 @@ export default class Calendar extends React.Component {
       var monthDate = addMonths(cloneDate(this.state.date), i)
       var monthKey = `month-${i}`
       monthList.push(
-        <div key={monthKey} ref={div => { this.monthContainer = div }} className="react-datepicker__month-container">
-          <div className="react-datepicker__header">
+        <div key={monthKey} ref={div => { this.monthContainer = div }} className='react-datepicker-month-container'>
```

Comment:
Binding is missing. This should be className={cx('react-datepicker-month-container')}